R/sprinkle.R

```diff
@@ -1,4 +1,4 @@
-#' @name sprinkles
+#' @name sprinkle
```

Comment:
There's nothing wrong with it, except it's redundant, and whenever you have the same information duplicated in two places you'll have to remember to keep them both updated.